modules/aatm/module.json

```diff
@@ -20,7 +20,7 @@
 			"twitter": "@ForgeVTT"
 		}
 	],
-	"url": "https://www.dndbeyond.com/sources/aatm",
+	"url": "https://www.dndbeyond.com/sources/dnd/aatm",
```

Comment:
This URL does appear to be valid, but redirects to `https://www.dndbeyond.com/sources/dnd/aatm/the-mortuary`